packages/oauth/src/index.ts

```diff
@@ -817,7 +823,7 @@ export interface OAuthV2TokenRefreshResponse extends WebAPICallResult {
   team: { id: string, name: string };
   enterprise: { name: string, id: string } | null;
   is_enterprise_install: boolean;
-  response_metadata: {}; // TODO
+  response_metadata: Record<string, never>; // TODO
```

Comment:
Is there a reason to keep this comment in this line now?

Reply:
Good question! I think so but am not sure - `Record` seems like a barely-more-acceptable type than `{}`?

Reply:
```suggestion
  response_metadata: Record<string, never>; // TODO revisit this type to see whether there are better alternatives
```

Reply:
Sounds good, I have removed it in the most recent commit!